pennylane/devices/default_qubit.py

```diff
@@ -208,6 +215,9 @@ def apply(self, operations, rotations=None, **kwargs):
                     f"on a {self.short_name} device."
                 )
 
+            if any(w in self._measured for w in operation.wires):
+                raise DeviceError("Cannot perform operation on measured wires.")
```

Comment:
Do we also need to preclude terminal measurements on a wire that has been mid-circuit-measured?

Reply:
As a follow up, would the plan be to add a transform that permits subsequent operations on measured qubits by mapping the subsequent operations to a separate wire?